src/OpenTelemetry/OpenTelemetry.csproj

```diff
@@ -3,9 +3,6 @@
     <TargetFrameworks>$(DefaultTargetFrameworks);netstandard2.1</TargetFrameworks>
     <Description>OpenTelemetry .NET SDK</Description>
     <MinVerTagPrefix>core-</MinVerTagPrefix>
-
-    <!-- this is temporary. will remove in future PR. -->
-    <Nullable>disable</Nullable>
```

Comment:
🥂 🚀 🥳

Reply:
Please check if you can remove `#nullable enable` from all files in this projec (the alternative approach is to do it, when all projects in repository will have nullable enabled).

Reply:
I'm going to do that as a follow-up so I don't explode the diff. Originally I wasn't going to enable at the project-level as part of this PR but I decided that is the most important part in order to stop the bleeding of code being added without annotations 😄